src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/PackagingTest.cs

```diff
@@ -56,6 +56,31 @@ public void CheckR8InfoMessagesToNotBreakTheBuild ()
 			}
 		}
 
+		[Test]
+		public void CheckDebugModeWithTrimming ()
+		{
+			bool usesAssemblyStores = false;
```

Comment:
Did you want to make this a parameter? The value never changes from `false`.

Reply:
The test didn't work with `true`, it doesn't seem to find any assembly stores. This could be another bug? But I manually checked and saw the assemblies inside.

Reply:
It doesn't find any assembly stores because there are no assemblies in the build (the `_Shrunk*` item groups are empty)

Reply:
This change fixes the `@(_Shrunk*)` item groups, I see them inside the `_PrepareAssemblies` target now:

![image](https://github.com/user-attachments/assets/f2a47572-86ea-47ee-8335-2633feb25f3f)

Reply:
Must be with FastDev enabled, because without it I don't see the `_Shrunk*` groups populated :(